test.disabled/ejabberd_tests/tests/assertions.hrl

```diff
@@ -20,3 +20,9 @@
                 erlang:error({assertReceivedMatch_failed, __Args__})
         end
       end)())).
+
+-define(assert_equal(E, V), (
```

Comment:
How is that different from EUnit's `assert`?

Reply:
This macro does not seem to be used anywhere in the changeset